modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -0,0 +1,5471 @@
+
+*Requires restart:* No
+
+*Optional:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17179869184`, `17179869183`]
+
+*Default:* `24h * 7`
+
+---
+
+=== group_topic_partitions
+
+Number of partitions in the internal group membership topic.
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-2147483648`, `2147483647`]
+
+*Default:* `16`
+
+---
+
+=== health_manager_tick_interval
+
+How often the health manager runs.
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `3min`
+
+---
+
+=== health_monitor_max_metadata_age
+
+Maximum age of the metadata cached in the health monitor of a non-controller broker.
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `10s`
+
+---
+
+=== http_authentication
+
+A list of supported HTTP authentication mechanisms. `BASIC` and `OIDC` are allowed.
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `user`
+
+*Type:* array
+
+*Default:* `[basic]`
+
+---
+
+=== id_allocator_batch_size
+
+The ID allocator allocates messages in batches (each batch is a one log record) and then serves requests from memory without touching the log until the batch is exhausted.
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-32768`, `32767`]
+
+*Default:* `1000`
+
+---
+
+=== id_allocator_log_capacity
+
+Capacity of the `id_allocator` log in number of batches. After it reaches `id_allocator_stm`, it truncates the log's prefix.
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-32768`, `32767`]
+
+*Default:* `100`
+
+---
+
+=== initial_retention_local_target_bytes_default
+
+Initial local retention size target for partitions of topics with xref:manage:tiered-storage.adoc[Tiered Storage] enabled.
+
+*Units*: bytes.
+
+*Requires restart:* No
+
+*Optional:* No
+
+*Visibility:* `user`
+
+*Type:* integer
+
+*Default:* `null`
+
+---
+
+=== initial_retention_local_target_ms_default
+
+Initial local retention time target for partitions of topics with xref:manage:tiered-storage.adoc[Tiered Storage] enabled.
+
+*Units*: milliseconds
+
+*Requires restart:* No
+
+*Optional:* No
+
+*Visibility:* `user`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `null`
+
+---
+
+=== internal_topic_replication_factor
+
+Target replication factor for internal topics.
+
+*Units*: number of replicas per topic.
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `user`
+
+*Type:* integer
+
+*Accepted values:* [`-2147483648`, `2147483647`]
+
+*Default:* `3`
+
+---
+
+=== join_retry_timeout_ms
+
+Time between cluster join retries in milliseconds.
```

Comment:
When we say something is specified in ms, why do we not use that unit when showing the default?